lgsm/data/serverlist.csv

```diff
@@ -3,6 +3,7 @@ ahl,ahlserver,Action Half-Life
 ahl2,ahl2server,Action: Source
 ark,arkserver,ARK: Survival Evolved
 arma3,arma3server,ARMA 3
+armareforger,armareforgerserver,ARMA Reforger
```

Comment:
Not sure about this.
@dgibbs64 do you think the full name should be used here ?